src/main/java/com/moabam/api/application/coupon/CouponService.java

```diff
@@ -59,24 +61,33 @@ public CouponResponse getById(Long couponId) {
 		Coupon coupon = couponRepository.findById(couponId)
 			.orElseThrow(() -> new NotFoundException(ErrorMessage.NOT_FOUND_COUPON));
 
-		return CouponMapper.toDto(coupon);
-	}
-
-	public Coupon getByWalletIdAndMemberId(Long couponWalletId, Long memberId) {
-		return couponWalletSearchRepository.findByIdAndMemberId(couponWalletId, memberId)
-			.orElseThrow(() -> new NotFoundException(ErrorMessage.NOT_FOUND_COUPON_WALLET))
-			.getCoupon();
+		return CouponMapper.toResponse(coupon);
 	}
 
 	public List<CouponResponse> getAllByStatus(CouponStatusRequest request) {
 		LocalDate now = clockHolder.date();
 		List<Coupon> coupons = couponSearchRepository.findAllByStatus(now, request);
 
 		return coupons.stream()
-			.map(CouponMapper::toDto)
+			.map(CouponMapper::toResponse)
+			.toList();
```

Comment:
```suggestion
		return StreamUtils.map(coupons, CouponMapper::toResponse);
```
C: `.stream().map(CouponMapper::toResponse).toList();` 요런 형태가 자주 쓰여서 StreamUtil 만들어놨었음! 괜찮으면 적용해도 좋을듯여

Reply:
헉!! 그렇군요 바로 적용하겠음다